scripts/lvi-mitigation/clang-7-lvi

```diff
@@ -2,33 +2,14 @@
 # Copyright (c) Open Enclave SDK contributors.
 # Licensed under the MIT License.
```

Comment:
Part of these compiler changes will change the underlying expected environment, as these bits are linked in with the ansible on rebuilds. Can you ping me when you are ready to merge and we can run an e2e to ensure nothing has changed unexpectedly?

Reply:
Sure. Also, I didn't include changes to ansible this PR, which may be required and  I'm thinking of opening another follow-up one after this one is merged. What do you think?

Note that this PR does not impact the build process of OE (and the tests), but only allow users to have more choices of compilers when building their enclave apps.